docs/README.md

```diff
@@ -157,6 +157,42 @@ aws_access_key_id = <YOUR ACCESS KEY ID>
 aws_secret_access_key = <YOUR SECRET ACCESS KEY>
 ```
 
+##### Multiple AWS Accounts
+
+In case you are using multiple AWS accounts and not depending on the default profile,
+you will need to add a `region` field to let AWS know which region we should deploy our application to.
+To do so, we have two possible solutions:
+* Adding the region to the profile in the `credentials` file
+
+```ini
+# ~/.aws/credentials
+[default]
+aws_access_key_id = <YOUR ACCESS KEY ID>
+aws_secret_access_key = <YOUR SECRET ACCESS KEY>
+region=<Region you want to deploy the application to>
+
+[other_profile]
+aws_access_key_id = <YOUR ACCESS KEY ID>
+aws_secret_access_key = <YOUR SECRET ACCESS KEY>
+region=<Region you want to deploy the application to>
```

Comment:
Let's be clearer, if we use the same placeholders in both profiles, some people might thing they need to set their Booster credentials in both places.
```suggestion
# ~/.aws/credentials
[default]
aws_access_key_id = <DEFAULT ACCESS KEY ID>
aws_secret_access_key = <DEFAULT SECRET ACCESS KEY>
region=<DEFAULT REGION>

[other_profile] # Give this profile the name that works best for you
aws_access_key_id = <YOUR ACCESS KEY ID>
aws_secret_access_key = <YOUR SECRET ACCESS KEY>
region=<REGION FOR YOUR BOOSTER APP>
```